core/ui/public/i18n/en/translation.json

```
@@ -575,6 +584,31 @@
     "settings_cluster": {
         "title": "Cluster"
     },
+    "settings_subscription": {
+        "title": "Subscription",
+        "cluster_subscription": "Cluster subscription",
+        "cluster_subscription_description": "The cluster subscription is used to enable the cluster features and to receive updates for the cluster nodes.",
+        "subscription_status": "Subscription status",
+        "authentication_token": "Authentication token",
+        "authentication_token_helper": "The authentication token is used to authenticate the cluster nodes to the subscription service.",
+        "request_subscription": "Request",
+        "remove_subscription": "Remove",
+        "authentication_token_placeholder": "Paste the authentication token",
+        "system_id": "System ID",
+        "plan_name": "Plan name",
+        "expire_date": "Expiration",
+        "status": "Status",
+        "remote_support": "Remote support",
+        "remote_support_description": "Remote support is used to allow Nethesis support team to access your cluster nodes for troubleshooting purposes.",
```

```suggestion
        "remote_support_description": "Remote support allows {productName} support team to access your cluster for troubleshooting purposes.",
```
Then let's pass `productName` to the translation function